6212a2c5f822a575b3da6baaf26b3d282801c759
src/test/java/com/yelp/nrtsearch/server/luceneserver/highlights/NRTFastVectorHighlighterTest.java
--- 
+++ 
@@ -433,7 +433,7 @@
                                     .setField("comment_multivalue")
                                     .setQuery("food")))
                     .setMaxNumberOfFragments(UInt32Value.of(2))
-                    .setFragmentSize(UInt32Value.of(Integer.MAX_VALUE))
+                    .setFragmentSize(UInt32Value.of(0))
                     .setDiscreteMultivalue(BoolValue.of(true))
                     .setFieldMatch(BoolValue.of(true)))
             .addFields("comment_multivalue")